tests/test_ao_citations.py

```diff
@@ -117,3 +118,18 @@ def test_validate_regulation_citation(title, part, expected):
 ])
 def test_parse_regulatory_citations(text, expected):
     assert parse_regulatory_citations(text) == expected
+
+
+class TestManualCitations(unittest.TestCase):
+    def test_exclude_citations(self):
+        original_ao_citations = set(['2011-12', '2017-03', '2014-18', '2010-11', '2004-41', '2007-13', '1996-26', '2005-17', '2017-01', '2016-02', '2012-21', '2012-23', '2014-11', '2002-15', '2006-12', '2015-16', '2014-21'])
```

Comment:
The only comment I would make is that the test data is much shorter so that it is easy to tell at a glance the difference between the original and expected citations.

Reply:
Thanks so much for taking a look! I had it that way before and couldn’t make up my mind - I agree that it was much easier to read. I will change it back :)